pkg/templates/cloudwatch-agent-config.json

```diff
@@ -0,0 +1,30 @@
+{
+	"agent": {
+		"metrics_collection_interval": 30,
+		"run_as_user": "root"
+	},
+	"metrics": {
+		"append_dimensions": {
+			"AutoScalingGroupName": "${aws:AutoScalingGroupName}",
+			"InstanceId": "${aws:InstanceId}",
+			"InstanceType": "${aws:InstanceType}"
```

Comment:
maybe a dimension for the test run would be good too

Reply:
Agreed, but from the [docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-Configuration-File-Details.html):


append_dimensions – Optional. Adds Amazon EC2 metric dimensions to all metrics collected by the agent. **The only supported key-value pairs are shown in the following list. Any other key-value pairs are ignored.**

"ImageID":"${aws:ImageId}" sets the instance's AMI ID as the value of the ImageID dimension.

"InstanceId":"${aws:InstanceId}" sets the instance's instance ID as the value of the InstanceID dimension.

"InstanceType":"${aws:InstanceType}" sets the instance's instance type as the value of the InstanceType dimension.

"AutoScalingGroupName":"${aws:AutoScalingGroupName}" sets the instance's Auto Scaling group name as the value of the AutoScalingGroupName dimension.